sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentLanguage.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics.Models
+{
+    internal partial class DocumentLanguage
+    {
+        internal static DocumentLanguage DeserializeDocumentLanguage(JsonElement element)
```

Comment:
Why did you have to copy this?

Reply:
Because [DetectedLanguage](https://github.com/Azure/azure-sdk-for-net/blob/e0a01b608703cb1ecde68e7f90c60358af566fe0/sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentLanguage.cs#L47) needs the warnings, so after I read the warnings I am creating the `detectedLanguage` object.